sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cas-integration/_tests_/cas-supplier.factory.ts

```diff
@@ -0,0 +1,16 @@
+import { CASSupplier, Student, SupplierStatus, User } from "@sims/sims-db";
+import { E2EDataSources } from "@sims/test-utils";
+
+export async function saveFakeCASSupplier(
+  db: E2EDataSources,
+  auditUser: User,
+  student: Student,
+) {
+  const casSupplier = new CASSupplier();
+  casSupplier.supplierStatus = SupplierStatus.PendingSupplierVerification;
+  casSupplier.supplierStatusUpdatedOn = new Date();
+  casSupplier.isValid = false;
+  casSupplier.creator = auditUser;
+  casSupplier.student = student;
+  return db.casSupplier.save(casSupplier);
+}
```

Comment:
Please check some other factories to follow similar approaches.
- Add comments.
- Relations go under a relations object.
- Student can be created if not provided.
- Missing return type.

The PR comment that probably motivated the creation of this method was requesting the creation of the `createFake` only. The `saveFake` would be a good addition but was not a blocker.